web_development_101/project_html_css.md

```diff
@@ -68,1128 +68,1134 @@ Note: All the classes and id's and names of elements that you inspect on Google'
 
 #### Option 1: github-pages
 
-Github allows you to publish web projects directly from a github repository.  Doing this will allow you to access your project from `your-gh-username.github.io/your-github-repo-name`
+Github allows you to publish web projects directly from a GitHub repository.  Doing this will allow you to access your project from `your-gh-username.github.io/your-github-repo-name`
 
 There are a couple of ways to go about doing this, but the simplest is this:
 
 - make sure that the main html file of your project is called `index.html`.  If it is not, you will need to rename it.
-- go to your github repo on the web and click the __Settings__ button
+- go to your GitHub repo on the web and click the __Settings__ button
 - scroll down until you find the __GitHub Pages__ section
 - change the __Source__ from _none_ to _master branch_ and click Save.
 - it may take a few minutes (the GitHub website says up to 10) but your project should be accessible over the web from `your-gh-username.github.io/your-github-repo-name` (obviously substituting your own details in the link)
 
 #### Option 2: rawgit
 
-[Rawgit.com](https://rawgit.com) is another option for viewing github projects online.  It is simpler and quicker, but less permanent, likely less reliable than gh-pages, and you don't get the benefit of a nice personalized URL.  That said, for just quickly sending someone a preview of something you aren't ready to officially publish, it works great.
+[Rawgit.com](https://rawgit.com) is another option for viewing GitHub projects online.  It is simpler and quicker, but less permanent, likely less reliable than gh-pages, and you don't get the benefit of a nice personalized URL.  That said, for just quickly sending someone a preview of something you aren't ready to officially publish, it works great.
 
 - go to the website: [Rawgit.com](https://rawgit.com)
-- paste in the url to the main html file from your github repo (probably `index.html`)
+- paste in the url to the main html file from your GitHub repo (probably `index.html`)
 - copy and use one of the supplied links to share your project.
 
-### Student Solutions
-Submit a link below to the github repo with your files in it by using a pull request.  See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.
+### Student solutions
+Submit a link below to the GitHub repo with your files in it by using a pull request.  See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.
 
-* Add your solution below this list
+**Add your solution below this list:**
```

Comment:
This is the style of the `Add your solution below this list` on every page on the site. It would be better left as it was so it matches.